2024-09-20 17:24:38,762 INFO    MainThread:43508 [wandb_setup.py:_flush():77] Current SDK version is 0.18.0
2024-09-20 17:24:38,762 INFO    MainThread:43508 [wandb_setup.py:_flush():77] Configure stats pid to 43508
2024-09-20 17:24:38,762 INFO    MainThread:43508 [wandb_setup.py:_flush():77] Loading settings from C:\Users\LWJ\.config\wandb\settings
2024-09-20 17:24:38,762 INFO    MainThread:43508 [wandb_setup.py:_flush():77] Loading settings from F:\Repository\rl_algorithm\wandb\settings
2024-09-20 17:24:38,762 INFO    MainThread:43508 [wandb_setup.py:_flush():77] Loading settings from environment variables: {}
2024-09-20 17:24:38,762 INFO    MainThread:43508 [wandb_setup.py:_flush():77] Applying setup settings: {'_disable_service': False}
2024-09-20 17:24:38,763 INFO    MainThread:43508 [wandb_setup.py:_flush():77] Inferring run settings from compute environment: {'program_relpath': 'ppo_main.py', 'program_abspath': 'f:\\Repository\\rl_algorithm\\ppo_main.py', 'program': 'f:\\Repository\\rl_algorithm\\ppo_main.py'}
2024-09-20 17:24:38,763 INFO    MainThread:43508 [wandb_setup.py:_flush():77] Applying login settings: {}
2024-09-20 17:24:38,763 INFO    MainThread:43508 [wandb_init.py:_log_setup():525] Logging user logs to F:\Repository\rl_algorithm\wandb\run-20240920_172438-aqzyoxkd\logs\debug.log
2024-09-20 17:24:38,763 INFO    MainThread:43508 [wandb_init.py:_log_setup():526] Logging internal logs to F:\Repository\rl_algorithm\wandb\run-20240920_172438-aqzyoxkd\logs\debug-internal.log
2024-09-20 17:24:38,763 INFO    MainThread:43508 [wandb_init.py:init():609] calling init triggers
2024-09-20 17:24:38,763 INFO    MainThread:43508 [wandb_init.py:init():616] wandb.init called with sweep_config: {}
config: {}
2024-09-20 17:24:38,763 INFO    MainThread:43508 [wandb_init.py:init():659] starting backend
2024-09-20 17:24:38,763 INFO    MainThread:43508 [wandb_init.py:init():663] setting up manager
2024-09-20 17:24:38,764 INFO    MainThread:43508 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-09-20 17:24:38,766 INFO    MainThread:43508 [wandb_init.py:init():671] backend started and connected
2024-09-20 17:24:38,768 INFO    MainThread:43508 [wandb_init.py:init():766] updated telemetry
2024-09-20 17:24:38,962 INFO    MainThread:43508 [wandb_init.py:init():799] communicating run to backend with 90.0 second timeout
2024-09-20 17:24:41,977 WARNING MainThread:43508 [wandb_init.py:init():1238] interrupted
Traceback (most recent call last):
  File "C:\Users\LWJ\AppData\Roaming\Python\Python39\site-packages\wandb\sdk\wandb_init.py", line 1234, in init
    return wi.init()
  File "C:\Users\LWJ\AppData\Roaming\Python\Python39\site-packages\wandb\sdk\wandb_init.py", line 802, in init
    result = run_init_handle.wait(
  File "C:\Users\LWJ\AppData\Roaming\Python\Python39\site-packages\wandb\sdk\lib\mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "C:\Users\LWJ\AppData\Roaming\Python\Python39\site-packages\wandb\sdk\lib\mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "C:\Users\LWJ\AppData\Roaming\Python\Python39\site-packages\wandb\sdk\lib\mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "D:\Anaconda\envs\py39\lib\threading.py", line 581, in wait
    signaled = self._cond.wait(timeout)
  File "D:\Anaconda\envs\py39\lib\threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-09-20 17:24:42,010 WARNING MsgRouterThr:43508 [router.py:message_loop():77] message_loop has been closed
